src/main/java/frc/robot/Constants.java

```diff
@@ -48,120 +40,30 @@ public static final class DrivetrainConstants {
          */
 
         // Choose the slower motor speed max, in this case the right motor
-        public static final double MAX_ENCODER_VELOCITY = 20743.0d;
-        public static final double KF_TYPICAL_PERCENT_USAGE = 0.75d; // We will typically use 75% of max speed
-        public static final double TALON_FULL_OUTPUT_SETTING = 1023;
-        public static final double VELOCITY_KF = 0.046d;
-        public static final double VELOCITY_KP = 0.03d;
-        public static final double VELOCITY_KI = 0.0d;
-        public static final double VELOCITY_KD = 0.06d;
-
-        public static final double POSITION_KF = 0.0d;
-        public static final double POSITION_KP = 0.029d;
-        public static final double POSITION_KI = 0.0004d;
-        public static final double POSITION_KD = 0.29d;
-
-        public static final double CLOSED_LOOP_RAMP = 0.5;
-        public static final double MAX_VELOCITY = 21549;
-
-        public static final double DEFAULT_MAX_VELOCITY_PERCENTAGE = 0.85;
-        public static final double DEFAULT_MAX_TURNING_SPEED = 0.55d;
-        public static final double VELOCITY_SLOWDOWN_MODIFIER = 0.25d;
-        public static final int LEFT_LEAD_TALON_CAN_ID = 2;
-        public static final int LEFT_FOLLOWER_TALON_CAN_ID = 0;
-        public static final int RIGHT_LEAD_TALON_CAN_ID = 1;
-        public static final int RIGHT_FOLLOWER_TALON_CAN_ID = 3;
-        public static final int VELOCITY_PID_SLOT_ID = 0;
-        public static final int POSITION_PID_SLOT_ID = 1;
-        public static final int PID_CONFIG_TIMEOUT_MS = 10;
-        public static final int CONFIG_FEEDBACKSENSOR_TIMEOUT_MS = 4000;
-        public static final double MOTOR_NEUTRAL_DEADBAND = 0.001d;
-        public static final int ALLOWABLE_DRIVETRAIN_ERROR = 200;
-
         // MISC Constants
-        public static final double WHEEL_CIRCUMFERENCE_FEET = (4.0d / 12.0d) * Math.PI; // Wheel radius 4 in, converting
-                                                                                        // to feet
-        public static final double SECONDS_TO_DECISEC = 1.0d / 10.0d;
-        public static final double DECISEC_TO_SECONDS = 10.0d / 1.0d;
-        public static final double GEARBOX_RATIO_TO_ONE = 9.52d;
-        public static final int ENCODER_COUNTS_PER_REVOLUTION = 2048;
-        public static final int ENCODER_EDGES_PER_STEP = 1;
-        public static final int ENCODER_UNITS_PER_REVOLUTION = ENCODER_COUNTS_PER_REVOLUTION; // Edges per Rotation
+        // kinematics - driveto - drivetrain
         public static final double TRACK_WIDTH_FEET = 27.5d / 12.0d; // Track width is 27.5 inches
```

Comment:
This can be moved into drivetrain.java and outside subsystems can get it by calling `drivetrain.getTrackWidthFeet()` which you'll have to make